src/http/content.cr

```diff
@@ -26,18 +60,22 @@ module HTTP
     end
 
     def read(slice : Bytes)
+      ensure_send_continue
       @io.read(slice)
     end
 
     def read_byte
+      ensure_send_continue
       @io.read_byte
     end
 
     def peek
+      ensure_send_continue
       @io.peek
     end
 
     def skip(bytes_count)
+      enensure_send_continue
```

Comment:
One `en` too much.

Reply:
... i don't... even..

Reply:
Relax, that happens, especially when there's lack of sleep included :) I'd just advise "proofreading" your PRs before submitting ;)